server/src/main/java/org/elasticsearch/action/index/IndexRequest.java

```diff
@@ -738,7 +737,7 @@ private void writeBody(StreamOutput out) throws IOException {
             out.writeMap(dynamicTemplates, StreamOutput::writeString);
         } else {
             if (dynamicTemplates.isEmpty() == false) {
-                throw new IllegalArgumentException("[dynamic_templates] parameter requires all nodes on " + Version.V_7_13_0 + " or later");
+                throw new IllegalArgumentException("[dynamic_templates] parameter requires all nodes on 7.13.0 or later");
```

Comment:
This should be referencing the transport version, not the node version

Reply:
They should coincide for 7.13, right?
Maybe we can change the message, but isn't more clear to refer to node (or ES) version?

Reply:
This can use the version mapping code now, to output release version or transport version as appropriate

Reply:
The utility here is limited, because it is a constant anyway, but it's nice to be consistent, so I have updated it!

Reply:
@thecoop Updated to use the instance version (`toReleaseVersion`)